pkgs/top-level/release-cuda.nix

```diff
@@ -1,60 +1,41 @@
 /*
     Test CUDA packages.
 
-    This release file will not be tested on hydra.nixos.org
-    because it requires unfree software.
+    This release file is currently not tested on hydra.nixos.org
+    because it requires unfree software, but it is tested by
+    https://hydra.nix-community.org/jobset/nixpkgs/cuda-nixos-unstable.
 
-    Test for example like this:
+    Cf. https://github.com/nix-community/infra/pull/1335
 
-        $ hydra-eval-jobs pkgs/top-level/release-cuda.nix --option restrict-eval false -I foo=. --arg nixpkgs '{ outPath = ./.; revCount = 0; shortRev = "aabbcc"; }'
+    Test for example like this:
 
+        $ hydra-eval-jobs pkgs/top-level/release-cuda.nix -I .
 */
 
-{ # The platforms for which we build Nixpkgs.
+{
+  # The platforms for which we build Nixpkgs.
   supportedSystems ? [
     "x86_64-linux"
-  ]
-, # Attributes passed to nixpkgs.
-  nixpkgsArgs ? { config = { allowUnfree = true; inHydra = true; }; }
+    "aarch64-linux"
+  ],
+  # Attributes passed to nixpkgs.
+  nixpkgsArgs ? {
+    config = {
+      cudaSupport = true;
+      allowUnfree = true;
+      inHydra = true;
+    };
+  },
 }:
 
 let
-  release-lib = import ./release-lib.nix {
-    inherit supportedSystems nixpkgsArgs;
+  release-lib = import ./release-lib.nix { inherit supportedSystems nixpkgsArgs; };
+
+  inherit (release-lib) mapTestOn packagePlatforms pkgs;
+
+  jobs = {
+    pkgs = mapTestOn (packagePlatforms pkgs);
```

Comment:
I'm not familiar with `release-lib.nix` and hydra. Is there any reason not to do _this_?

Reply:
Picked a smaller subset now